docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,119 @@
+
+### Automation
+
+[[= product_name_cloud =]] automates time-consuming testing processes and encourages ongoing QA testing, so your projects are ready for deployment much faster.
+For further protection and simple rollbacks, it also automates your backup procedures.
+This guarantees that the developers are working within a framework that is adaptable, safe, and responsive to their requirements.
+Thanks to built-in CI/CD features, you can also reduce the need for manual testing and accelerate your development.
+
+![Ibexa Cloud - effort](ibexa_cloud_effort.png)
+
+### Digital tool
+
+[[= product_name_cloud =]] is a digital tool - it's a perfect starting step toward your company's digital transformation.
+What's more, [[= product_name_cloud =]] as an end-to-end cloud hosting platform, it's scalable, requires little up-front expenditure, and no ongoing maintenance.
+[[= product_name_cloud =]] as a PaaS solution enables you to start constructing effective digital systems that provide great consumer experiences.
+
+## Benefits
+
+### Safety and simplicity
+
+Comprehensive data security procedures guarantee that you maintain complete ownership of your client's data, defining where it's stored.
+You can be sure that the safety protocols are compliant with all applicable legislation.
+What's more, all updates to your code and infrastructure are fully auditable. Global CDN (Content Delivery Network) is included and fully managed.
+[[= product_name_cloud =]], thanks to grid architecture and extensive [[= product_name_base =]] support, enables effortless deployment.
+You can create a clone not only of the code, but also data and the infrastructure.
+As the infrastructure is exactly the same as what's currently in production, you can be sure that everything works well when you conduct your release and push it live.
+
+### High availability and compatibility
+
+[[= product_name_cloud =]] is compatible with your choice of public cloud server and supports a variety of hosting platforms.
+It's a Git-native development - compatible with Git Flow. [[= product_name_cloud =]] deployment integrates naturally at the end of your existing production chain, including staging, and work in progress branch preview.
+You can also integrate with, for example, Bitbucket, GitHub, Gitlab.
+You can instantly clone every branch of both your code and infrastructure configuration.
+Platform.sh (and [[= product_name_cloud =]], by extension) uses the [Infrastracture as Code approach](https://docs.platform.sh/learn/overview.html#infrastructure-as-code).
+It means that the infrastracture is described in the code and that is what allows you to clone both code and infrastracture configuration at the same time.
+If you want to work with services such as [MySQL](https://docs.platform.sh/add-services/mysql.html), [Kafka](https://docs.platform.sh/add-services/kafka.html), and [Elasticsearch](https://docs.platform.sh/add-services/elasticsearch.html), you can add them with a line of code.
```

Comment:
I don't think we support Kafka... I suggest to remove it

Reply:
Comes from: https://www.ibexa.co/products/ibexa-cloud

<img width="576" alt="image" src="https://github.com/user-attachments/assets/bf6d744f-a649-4cd9-8864-361a5d03b819">

Reply:
Hummm then we need to tell marketing... :D

Reply:
Karl just removed it from ibexa.co website.